vllm/plugins/__init__.py

```diff
@@ -1,17 +1,51 @@
 import logging
 import os
+from typing import Callable, Dict
 
 import torch
 
 import vllm.envs as envs
-from vllm.platforms import current_platform
 
 logger = logging.getLogger(__name__)
 
 # make sure one process only loads plugins once
 plugins_loaded = False
 
 
+def load_plugins_by_group(
+        group: str = 'vllm.general_plugins') -> Dict[str, Callable]:
```

Comment:
I prefer explicitly passing in the group name each time.

Reply:
fixed in [c91fa49](https://github.com/vllm-project/vllm/pull/11602/commits/c91fa49de3c4fc8efe9d5e77dea086028ebe0572)